detekt-api/src/main/kotlin/io/gitlab/arturbosch/detekt/api/internal/Signatures.kt

```diff
@@ -1,16 +1,18 @@
 package io.gitlab.arturbosch.detekt.api.internal
 
+import io.github.detekt.psi.fileName
 import org.jetbrains.kotlin.asJava.namedUnwrappedElement
 import org.jetbrains.kotlin.com.intellij.psi.PsiElement
 import org.jetbrains.kotlin.psi.KtClassOrObject
 import org.jetbrains.kotlin.psi.KtFile
 import org.jetbrains.kotlin.psi.KtNamedFunction
 import org.jetbrains.kotlin.psi.psiUtil.endOffset
 import org.jetbrains.kotlin.psi.psiUtil.getNonStrictParentOfType
+import org.jetbrains.kotlin.psi.psiUtil.getStrictParentOfType
 import org.jetbrains.kotlin.psi.psiUtil.parents
 import org.jetbrains.kotlin.psi.psiUtil.startOffset
 
-private val signatureRegex = Regex("\\s(\\s|\t)+")
+private val multipleWhitespaces = Regex("\\s(\\s|\t)+")
```

Comment:
```suggestion
private val multipleWhitespaces = Regex("\\s{2,}")
```
This should suffice and is also more aligned to the name of the variable.